plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -188,8 +188,8 @@ instance Show (BuiltinRuntime (CekValue uni fun)) where
 data CekValue uni fun =
     -- This bang gave us a 1-2% speed-up at the time of writing.
     VCon !(Some (ValueOf uni))
-  | VDelay (Term NamedDeBruijn uni fun ()) !(CekValEnv uni fun)
-  | VLamAbs NamedDeBruijn (Term NamedDeBruijn uni fun ()) !(CekValEnv uni fun)
+  | VDelay !(Term NamedDeBruijn uni fun ()) !(CekValEnv uni fun)
+  | VLamAbs !NamedDeBruijn !(Term NamedDeBruijn uni fun ()) !(CekValEnv uni fun)
```

Comment:
The first bang might be making GHC unpack `NamedDeBruijn`, which isn't necessarily a win. Not sure.

Reply:
We don't enable `unbox-strict-fields` globally, right? And it's not enabled by default (unlike `unbox-small-strict-fields`). So I think that shouldn't happen? Gonna rerun the benchmarks.

Reply:
Wait, `NamedDeBruijn` isn't even strict? I see neither bangs no `StrictData` in [this](https://github.com/input-output-hk/plutus/blob/master/plutus-core/plutus-core/src/PlutusCore/DeBruijn/Internal.hs) file.

Reply:
I guess I'm just making things up then. I thought `unbox-small-strict-fields` might hit it too.